src/js/utils/DOM.js

```diff
@@ -67,7 +67,12 @@ export const containsFocus = (node) => {
 // Check if the element.tagName is an input, select or textarea
 export const isFocusable = (element) => {
   const tagName = element.tagName.toLowerCase();
-  return tagName === 'input' || tagName === 'select' || tagName === 'textarea';
+  return (
+    tagName === 'input' ||
```

Comment:
I think relying on tagName misses a lot of rich or custom components. For example a RangeSelector has two little controls, one of which may have the focus. Those controls are just `div`s with `tabindex="0"`. If a RangeSelector is somehow one of the descendents then it won't get counted. Focusable things include those tags plus basically anything with `tabindex` (even if it is "-1").

Reply:
Currently looking into if there is a better way to handle this

Reply:
I think adding the tabindex check OR'd with the above tags is promising.

Reply:
Updated in latest commit